resources/tickets.py

```diff
@@ -92,3 +92,19 @@ def post(self):
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def delete(self):
+        self.parser.add_argument("ids", action="append")
+        data = Tickets.parser.parse_args()
```

Comment:
The parser is being removed, let's not add to our tech debt. You can access the endpoints params with `request.json`

Reply:
Is there documentation on things we have deprecated but haven't replaced yet? I seem to keep using functions or patterns that are "deprecated" but are still used abundantly throughout the app

Reply:
Yes, thanks for asking. The CONTRIBUTING.md document